pyproject.toml

```
@@ -2,13 +2,137 @@
 requires = ["setuptools>=42", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "imod"
+version = "0.15.0"
+requires-python = ">=3.10"
+authors = [
+    { name = "Deltares", email = "huite.bootsma@deltares.nl" },
+]
+maintainers = [
+    { name = "Deltares", email = "huite.bootsma@deltares.nl" }
+]
+description = "Make massive MODFLOW models!"
+readme = "README.rst"
+license = { file = "LICENSE" }
+keywords = ["imod", "modflow", "groundwater modeling"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Hydrology",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

This is outdated info, should be:

```suggestion
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
```